server/utils/contestHelper.js

```diff
@@ -53,15 +54,40 @@ exports.scheduleContestEnd = asyncHandler(async (contest) => {
   }
 })
 
+const stripePay = asyncHandler(async (contestOwner, prizeAmount, winnerId) => {
+  const findContestUser = await User.findOne({_id: contestOwner})
+
+  console.log('findContestUser', findContestUser)
```

Comment:
Please remove and fix indentation/spacing :)

Reply:
removed and indented